PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -471,7 +471,7 @@ def nanoAOD_customizeMC(process):
 
 ###Customizations needed for Wmass analysis                                                                                                              
 ###increasing the precision of selected GenParticles.                                                                                                             
-def nanoGenWmassCustomize(process):
+def nanoGenWmassCustomize_minimal(process):
```

Comment:
Can you give it a more descriptive name? customizeGenLeptonPrecision for example

Reply:
Done.